sdk/v2/core/events.go

```diff
@@ -232,6 +232,13 @@ type EventsClient interface {
 	List(context.Context, *EventsSelector, *meta.ListOptions) (EventList, error)
 	// Get retrieves a single Event specified by its identifier.
 	Get(context.Context, string) (Event, error)
+	// Clones a pre-existing Event, removing the original's metadata and Worker
+	// config in the process. If the original Event references a Project by ID,
+	// one new Event will be created. Otherwise, the Event provided is treated
+	// as a template and zero or more discrete Events may be created-- one for
+	// each subscribed Project. An EventList is returned containing all newly
+	// created Events.
```

Comment:
By the time an event has an ID, it's a discrete event, so copy(eventID) can't possibly be copying a template. I think everything is good and just the comment here has to change.